commands/delete-database.go

```diff
@@ -15,81 +17,177 @@ func init() {
 	reset := cli.Command{
 		Name:      "delete",
 		Aliases:   []string{"delete-database"},
-		Usage:     "Delete an imported database",
+		Usage:     "Delete imported database(s)",
 		ArgsUsage: "<database>",
 		Flags: []cli.Flag{
 			forceFlag,
 			configFlag,
+			allFlag,
+			matchFlag,
+			regexFlag,
+			dryRunFlag,
 		},
-		Action: func(c *cli.Context) error {
-			res := resources.InitResources(c.String("config"))
-			db := c.Args().Get(0)
-			if db == "" {
-				return cli.NewExitError("Specify a database", -1)
-			}
-
-			return deleteDatabase(db, res, c.Bool("force"))
-
-		},
+		Action: deleteDatabase,
 	}
 
 	bootstrapCommands(reset)
 }
 
 //deleteDatabase deletes a target database
-func deleteDatabase(database string, res *resources.Resources, forceFlag bool) error {
+func deleteDatabase(c *cli.Context) error {
+	res := resources.InitResources(c.String("config"))
+
+	// Different command flags
+	match := c.String("match")
+	regex := c.String("regex")
+	bulk := c.Bool("all")
+	force := c.Bool("force")
+	dryRun := c.Bool("dry-run")
+	var names []string
+
+	if checkFlags((match != ""), (regex != ""), bulk) {
+		return cli.NewExitError("Please select a single bulk option", -1)
+	}
 
-	// get all database names
-	names, err := res.DB.Session.DatabaseNames()
+	if bulk {
+		dbs := res.MetaDB.GetDatabases()
+		copy(names, dbs)
+	} else if match != "" {
 
-	// check if database exists
-	dbExists := util.StringInSlice(database, names)
+		// Get DB list
+		dbs := res.MetaDB.GetDatabases()
 
-	// check if metadatabase record for database exists
-	mDBExists := util.StringInSlice(database, res.MetaDB.GetDatabases())
+		// Find dbs with matching names
+		for _, db := range dbs {
+			if strings.Contains(db, match) {
+				names = append(names, db)
+			}
+		}
 
-	// if no force flag, verify action with the user
-	if !forceFlag {
-		fmt.Print("Are you sure you want to delete database ", database, " [y/N] ")
+	} else if regex != "" {
 
-		read := bufio.NewReader(os.Stdin)
+		// Get DB list
+		dbs := res.MetaDB.GetDatabases()
 
-		response, _ := read.ReadString('\n')
+		// Compile regex, check if it's valid
+		regq, err := regexp.Compile(regex)
+		if err != nil {
+			return cli.NewExitError(err.Error, -1)
+		}
 
-		response = strings.ToLower(strings.TrimSpace(response))
-		if response == "y" || response == "yes" {
-			fmt.Println("Deleting database: ", database)
-		} else {
-			return cli.NewExitError("Database "+database+" was not deleted.", 0)
+		// Find dbs with regex query
+		for _, db := range dbs {
+			matched := regq.MatchString(db)
+			if matched {
+				names = append(names, db)
+			}
+		}
+
+	} else {
+		// Single database deletion
+		tgtDB := c.Args().Get(0)
+		// get all database names
+		dbs := res.MetaDB.GetDatabases()
+		if find(dbs, tgtDB) != len(dbs) {
```

Comment:
Can you use [`util.StringInSlice()`](https://github.com/activecm/rita/blob/master/util/util.go#L80) instead of find()?